Header header
sensor_msgs/Image TemplateImage
sensor_msgs/Image SourceImage
uint16 particle_number
uint16[] x_position_of_particle
uint16[] y_position_of_particle
